internal/scripts/installation/compile_MET_all.sh

```diff
@@ -430,17 +437,22 @@ if [ $COMPILE_G2CLIB -eq 1 ]; then
   rm -rf ${LIB_DIR}/g2clib/g2clib*
   tar -xf ${TAR_DIR}/g2clib*.tar -C ${LIB_DIR}/g2clib
   cd ${LIB_DIR}/g2clib/g2clib*
-  sed -i 's|INC=.*|INC=-I${LIB_DIR}/include -I${LIB_DIR}/include/jasper|g' makefile
+  # JHG Switch sed commands from using single quotes to double-quotes since
```

Comment:
Julie, please remove the comment lines which include `# JHG`. Those are just notes to myself I used at breadcrumbs to remind myself where I'd made changes.

Reply:
Thanks @JohnHalleyGotway.  I have removed the JHG comments.